src/org/parosproxy/paros/view/MainMenuBar.java

```diff
@@ -89,7 +91,9 @@
     private JMenu menuZapMode = null;
     private ButtonGroup menuZapModeGroup = null;
     private Map<Mode, JRadioButtonMenuItem> menuZapModeMap = null;
-    // ZAP: Added standard report menu
+
+
+	// ZAP: Added standard report menu

```

Comment:
Is this white space change really needed?

https://github.com/zaproxy/zaproxy/wiki/DevGuidelines#formatting

Reply:
My bad, will fix it

Reply:
Tks